config/routes.rb

```diff
@@ -19,6 +19,13 @@
     post "post/:post_id/stop-streaming" => "bot#stop_streaming_response"
     get "bot-username" => "bot#show_bot_username"
   end
+
+  scope module: :ai_bot, path: "/ai-bot/shared-ai-conversations" do
+    post "/" => "shared_ai_conversations#create"
+    delete "/:share_key" => "shared_ai_conversations#destroy"
+    get "/:share_key" => "shared_ai_conversations#show"
```

Comment:
Bit of a nitpick, but the URL is super long:

```
http://localhost:4200/discourse-ai/ai-bot/shared-ai-conversations/PHBmFKwiEOMd1g2p7NRbAg
```

Could be cool to have a short share URL:

```
http://localhost:4200/discourse-ai/shared/PHBmFKwiEOMd1g2p7NRbAg

OR

http://localhost:4200/discourse-ai/convo/PHBmFKwiEOMd1g2p7NRbAg
```